binja/krf.py

```diff
@@ -0,0 +1,130 @@
+import logging
```

Comment:
General nit: It'd be good to clean up some of these prints, and either remove them or replace them with an appropriately-leveled `log` call.